modules/lotameIdSystem.js

```diff
@@ -0,0 +1,178 @@
+/**
+ * This module adds LotameId to the User ID module
+ * The {@link module:modules/userId} module is required
+ * @module modules/lotameIdSystem
+ * @requires module:modules/userId
+ */
+import * as utils from '../src/utils.js';
+import { ajax } from '../src/ajax.js';
+import { submodule } from '../src/hook.js';
+import { getStorageManager } from '../src/storageManager.js';
+
+const KEY_LAST_UPDATE = '_lota_last';
+const KEY_ID = '_lota_pano';
+const KEY_EXPIRY = '_lota_pano_bundle';
+const PROFILE_ID_NAME = '_cc_id';
+const MODULE_NAME = 'lotameId';
+const NINE_MONTHS_IN_SECONDS = 23328000;
```

Comment:
Why not just make it nine months in milliseconds?

Reply:
no good reason, I think I copied it from lightning tag